public/index.html

```diff
@@ -4,86 +4,400 @@
     <meta charset="utf-8">
     <title>06-TShirts</title>
     <link rel="stylesheet" href="/app.css">
-    <link rel="stylesheet" href="/app.css">
   </head>
   <body>
 
-  <div class="outer-container">
+    <div class="outer-container">
 
-  <div class="container">
+    <div class="container">
 
-    <div class="row">
+      <nav class="row">
 
-      <div class="grid-item">
+    <div class="grid-item">
 
-        <div class="shop-item">
+      <div class="shop-item">
 
-          <div class="shirt-div">
-            <img src" https://raw.githubusercontent.com/TIY-TN-FEE-2016-spring/06-tshirts/master/public/shirt.png" alt="shirt">
-          </div>
+      <div class="shirt-div">
+        <img src="https://raw.githubusercontent.com/TIY-TN-FEE-2016-spring/06-tshirts/master/public/shirt.png">
+      </div>
 
-          <div class="shop-item-info">
-            <h2 class="h2 shop-item-title"></h2>
-            <p class="shop-item-description"></p>
-          </div>
+<div class="shop-item-info">
+<h2 class="h2 shop-item-title">T-Shirts</h2>
+  <p class="shop-item-description">Made of fresh All-American cotton<br>and some other stuff.</br></p>
+</div>
 
-          <ul class="shop-item-buttons">
-            <li class="shop-button">
-              <span class="fa fa-refresh"></span>
-          </ul>
+<ul class="shop-item-buttons">
+  <li class="shop-button">
+    <span class="fa fa-refresh"></span>
 
-            <li class="shop-button">
-              <span class="fa fa-heart-o"></span>
-            </li>
-            <li class="shop-button dropdown">
-              <span class="fa fa-square"></span>
+
+  <li class="shop-button">
+    <span class="fa fa-heart-o"></span>
+  </li>
+  <li class="shop-button dropdown square">
+    <span class="fa fa-square"></span>
+
+    <ul class="dropdown-options">
+      <li class="dropdown__options-items-shirt-color-green">
```

Comment:
To reduce duplication on the dropdown items, I would recommend two classes at once.

Instead of just having `dropdown__options-items-shirt-color-green` have two classes: `dropdown__options-items dropdown__options-items-shirt-color-green`.

This will allow the shared styling (borders, padding, background) to be rules on `dropdown__options-items` and then `dropdown__options-items-shirt-color-green` just changes the font color.